spec/game_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require './lib/game.rb'
+
+describe Game do
+  describe '#won?' do
+    it 'Check if there is a winner' do
+      expect(subject.won?('X')).not_to eq('TIE')
+    end
+  end
+
+  describe '#draw?' do
+    it 'Check if there is a draw' do
+      expect(subject.draw?(9)).to be_truthy
+    end
+  end
+
+  describe '#over?' do
+    it 'Checks if the game is over' do
+      expect(subject.over?(9, 9)).to be_truthy
+    end
+  end
```

Comment:
- [ ] You should add negative scenarios for those methods. For instance, a test for the #win when the game has not yet finished, or for if the game is a draw (in this way, you can know for sure if your methods behave the way you want for all the different cases).